group_vars/all/wireless_profiles.yml

```diff
@@ -42,41 +42,6 @@ wireless_profiles:
         legacy_rates: false
         country: DE
 
-    ifaces:
-      - mode: ap
-        ssid: berlin.freifunk.net
-        encryption: none
-        network: dhcp
-        radio: [11a_standard, 11g_standard]
-        ifname_hint: ff
-
-      - mode: ap
-        ssid: berlin.freifunk.net Encrypted
-        encryption: owe
-        network: dhcp
-        radio: [11a_standard, 11g_standard]
-        ifname_hint: ffowe
-        ieee80211w: 1
-
-      - mode: mesh
-        mesh_id: Mesh-Freifunk-Berlin
-        radio: [11a_standard, 11g_standard, 11a_mesh]
-        mcast_rate: 12000                              # inzwischen 16000?
```

Comment:
Maybe we should have a look at the mcast-rate sometime in the future, but that is not a blocking issue.